assets/training/finetune_acft_hf_nlp/components_v2/finetune/text_generation/spec.yaml

```diff
@@ -319,5 +319,6 @@ outputs:
   output_model:
     type: uri_folder
     description: Output dir to save the finetuned lora weights
+    mode: rw_mount
```

Comment:
Isn't _output_model_ already in _rw_mount_?

Reply:
It is in pipeline, I am just keeping in command component as well for consistency. Did the same for others.